libcudacxx/include/cuda/std/__atomic/types.h

```diff
@@ -13,6 +13,8 @@
 
 #include <cuda/std/detail/__config>
 
+#include "nv/detail/__target_macros"
+
```

Comment:
Please add it below at the right place
```suggestion
```

Reply:
... Some extension thought I needed that apparently. 😮‍💨